exercises/randomgenerator.js

```diff
@@ -1,20 +1,30 @@
 
 
 class RandomGenerator{
-	constructor(){ 
+	constructor(min, max){ 
 		//takes in the minimum and maximum values
+		this.min = min;
+		this.max = max;
+		if(!this.min && !this.max){
+			this.min = 1;
+			this.max = 10;	
+		}
 		//if they are not defined, it sets the minimum to 1, and the max to 10
 		//makes storage for the random number
+		this.randomNumber = 0;
 	}
 	generate(){
 		//make a random value between the minimum and maximum values
+		this.randomNumber = Math.floor(Math.random() * this.max+1 ) + this.min;
```

Comment:
this very well might give you an error because for, 0 to 20 for example, it would do 1 to 20 since the addition would happen after the multiplication
max = 20.  random of .99999 * 20 is 19.99999.  floored is 19.  plus 1 is 20
min = 0  random of .99999 * 0 is 0.  floored is 0.  plus 1 is 1
so 0 to 20 would give you 1 to 20